lib/osf-components/addon/components/registries/schema-block-renderer/single-select-input/template.hbs

```diff
@@ -0,0 +1,16 @@
+<FormControls
+    @changeset={{@changeset}}
+    @disabled={{@disabled}}
+    @shouldShowMessages={{@shouldShowMessages}}
+    as |form|
+>
+    <form.select
+        data-test-single-select-input
+        @id={{@schemaBlock.registrationResponseKey}}
```

Comment:
Same here, is `id` necessary?

Reply:
Important here if there's a label for this field.